modules/swagger-jaxrs2/src/main/java/io/swagger/jaxrs2/OperationParser.java

```diff
@@ -78,6 +78,10 @@
             parameterObject.setIn(parameter.in());
             isEmpty = false;
         }
+        if (StringUtils.isNotBlank(parameter.example())) {
+            parameterObject.setIn(parameter.example());
```

Comment:
I guess this is wrong, would be `parameterObject.setParameter` instead? also it means no tests current cover the case, and should be added

Reply:
Replaced by parameterObject.setExample and test added.